src/libraries/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.Experimental.cs

```diff
@@ -55,10 +55,12 @@ public abstract partial class AdvSimd : System.Runtime.Intrinsics.Arm.ArmBase
         public static System.Runtime.Intrinsics.Vector64<byte> Add(System.Runtime.Intrinsics.Vector64<byte> left, System.Runtime.Intrinsics.Vector64<byte> right) { throw null; }
         public static System.Runtime.Intrinsics.Vector64<short> Add(System.Runtime.Intrinsics.Vector64<short> left, System.Runtime.Intrinsics.Vector64<short> right) { throw null; }
         public static System.Runtime.Intrinsics.Vector64<int> Add(System.Runtime.Intrinsics.Vector64<int> left, System.Runtime.Intrinsics.Vector64<int> right) { throw null; }
+        public static System.Runtime.Intrinsics.Vector64<long> Add(System.Runtime.Intrinsics.Vector64<long> left, System.Runtime.Intrinsics.Vector64<long> right) { throw null; }
```

Comment:
@echesakovMSFT it seems new APIs are being exposed in this ref file. We want to ensure new APIs get properly documented in triple slash in their src definitions. I suspect this `*Experimental.cs` file might not be exposed to the public yet, so feel free to ignore my request if that's the case. If this code gets published officially though, please make sure the APIs gets documented, or open an issue to track that work item.

Reply:
Ref files are not normally documented. Its the implementation that is (which in this case lives in `S.P.Corelib` and has the appropriate `///` comment).

Reply:
Correct, no documentation in ref files.
I added the comment here because the bot is notifying us which PRs are adding new lines to ref files, which in most cases means a new API is being added.
Thanks for making sure the APIs are documented in their implementation.